scripts/release/main.go

```diff
@@ -26,42 +30,75 @@ const (
 )
 
 func main() {
-	logger := slog.Make(sloghuman.Sink(os.Stderr)).Leveled(slog.LevelDebug)
+	r := &releaseCommand{
+		fs:     afero.NewOsFs(),
```

Comment:
One thing is missing here, and that's ensuring fs is set to project root (will fix this in a while).